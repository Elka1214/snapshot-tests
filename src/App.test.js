import { render, screen } from "@testing-library/react";
import App from "./App";
import { Renderer } from "react-dom";

test("renders a snapshot", () => {
  const tree = renderer.create(<App />).toJSON();
  expect(tree).toMatchSnapshot();
});
