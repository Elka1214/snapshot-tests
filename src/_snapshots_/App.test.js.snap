exports[`renders a snapshot 1`] = `
<div
  className="App"
>
  <div
    className="card"
    style={
      Object {
        "margin": "auto",
        "width": "30%",
      }
    }
  >
    <img
      className="card-img-top"
    />
    <div
      className="card-body"
    >
      <div
        className="card-title h5"
      />
      <div
        className="card-subtitle h6"
      >
        (
        )
      </div>
      <p
        className="card-text"
      />
      <button
        className="btn btn-primary"
        disabled={false}
        type="button"
      >
        Go to Profile
      </button>
    </div>
  </div>
</div>
